app/src/main/res/values-ru/strings.xml

```diff
@@ -175,6 +175,7 @@
     <string name="pref_enable_acra">Отсылать отчеты о падениях</string>
     <string name="pref_page_transitions">Анимированные переходы страниц</string>
     <string name="pref_fullscreen">Полноэкранный режим</string>
+    <string name="pref_dual_page_split">Двойное разделение страниц (ALPHA)</string>
```

Comment:
Can you remove the changes in this file? They are from Tachi main and I am planning on keeping these the same. If you want to translate them I suggest doing it through Weblate https://tachiyomi.org/help/contribution/#translation